src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts

```diff
@@ -137,16 +138,18 @@ export class RbdListComponent extends ListWithDetails implements OnInit {
       click: () => this.deleteRbdModal(),
       name: this.actionLabels.DELETE,
       disable: (selection: CdTableSelection) =>
-        !this.selection.first() ||
-        !this.selection.hasSingleSelection ||
+        !selection.first() ||
+        !selection.hasSingleSelection ||
         this.hasClonedSnapshots(selection.first()),
       disableDesc: () => this.getDeleteDisableDesc()
     };
     const copyAction: CdTableAction = {
       permission: 'create',
       canBePrimary: (selection: CdTableSelection) => selection.hasSingleSelection,
       disable: (selection: CdTableSelection) =>
-        !selection.hasSingleSelection || selection.first().cdExecuting,
+        !selection.hasSingleSelection ||
+        selection.first().cdExecuting ||
+        selection.first().image_format === RBDImageFormat.V1,
```

Comment:
Nit: you should be able to copy a v1 image (to a v2 image)

Reply:
Then the error came from within dashboard implementation. Will check for exceptions there.

Reply:
I think it fails because it tries to create the destination image as v1 format: if the format is not explicitly specified the source image format is used [1].

@dillman I suppose we should change this to use v2 format if it is not specified via image options?

Another problem with `copy` I see here is that although we allow to specify the destination image format via image options in C/C++ API, we do not provide such possibility neither for python nor for CLI. So, right now the dashboard can't force the destination image v2 format in any way. Do we want to improve this or it is not worth doing taking the fromat v1 is depricated? (I vote for the later)

[1] https://github.com/ceph/ceph/blob/master/src/librbd/internal.cc#L1126

Reply:
Ouch -- that's no good. I am going to submit another commit to fix that `copy` defaulting to the source format since we really should never be creating v1 images any more.

Reply:
Thanks for the hint @trociny. So if @dillaman is going to fix this, I'll re-enable copy (it was kinda-gracefully failing before).

Reply:
@epuertat I've opened a PR here: https://github.com/ceph/ceph/pull/35025